server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/CustomMethodContract.scala

```diff
@@ -179,6 +250,80 @@ trait CustomMethodContract {
       .build
   }
 
+  @Test
+  def shouldSupportCustomTypeNameWithIntStateFromString(server: GuiceJamesServer): Unit = {
```

Comment:
I cannot see the difference between shouldSupportCustomTypeNameWithIntStateFromString and shouldSupportCustomTypeNameWithIntStateFromInt...